services/BasicTypes.proto

```diff
@@ -340,24 +340,49 @@ message CurrentAndNextFeeSchedule {
     FeeSchedule nextFeeSchedule = 2; // Contains next Fee Schedule
 }
 
+/* This contains the IP Address and the port representing an endpoint of a Node in a network */
+message NodeEndpoint {
+    string ipAddress = 1; // The IP address of the node
+    string port = 2; // The port of the node
+}
+
 /*
 The metadata for a Node – including IP Address, and the crypto account associated with the Node.
+
+Note :  This structure is intended to be used to parse 0.0.102, use NodeAddressForClients to parse 0.0.101.
+        Usage of NodeAddress to parse 0.0.101 is deprecated.
 */
 message NodeAddress {
-    bytes ipAddress = 1; // The ip address of the Node with separator & octets
-    int32 portno = 2; // The port number of the grpc server for the node
-    bytes memo = 3; // The memo field of the node (usage to store account ID is deprecated)
+    bytes ipAddress = 1 [deprecated=true]; // The ip address of the Node with separator & octets
+    int32 portno = 2 [deprecated=true]; // The port number of the grpc server for the node
+    bytes memo = 3 [deprecated=true]; // The memo field of the node (usage to store account ID is deprecated)
```

Comment:
Should add a comment that it's deprecated since`nodeAccountId` and `description` should be used instead.

Reply:
Although in practice we never populated the memo with a description. So maybe we can leave off the latter part.